pkgs/applications/science/physics/crystfel/default.nix

```diff
@@ -224,7 +219,7 @@ stdenv.mkDerivation rec {
     downloadPage = "https://www.desy.de/~twhite/crystfel/download.html";
     license = licenses.gpl3Plus;
     maintainers = with maintainers; [ pmiddend ];
-    platforms = [ "x86_64-linux" "x86_64-darwin" ];
+    platforms = [ "x86_64-linux" "x86_64-darwin" "aarch64-linux" "aarch64-darwin" ];
```

Comment:
Maybe just set unix?

Reply:
Yeah, seems to be the same, thanks for the hint.